frontend/packages/ceph-storage-plugin/src/components/dashboard-page/storage-dashboard/top-consumers-card/top-consumers-card-body.tsx

```diff
@@ -43,15 +43,38 @@ export const TopConsumersBody: React.FC<TopConsumerBodyProps> = React.memo(
     if (topConsumerStatsResult.length) {
       const maxCapacityConverted = getMaxCapacity(topConsumerStatsResult);
       const sortedResult = topConsumerStatsResult.sort(sortResources);
-      const legends = topConsumerStatsResult.map((resource) => ({
-        name: getMetricType(resource, metricType),
-      }));
 
       const chartData = getGraphVectorStats(sortedResult, metricType, maxCapacityConverted.unit);
 
       const chartLineList = chartData.map((data, i) => (
         <ChartLine key={i} data={data as DataPoint[]} /> // eslint-disable-line react/no-array-index-key
       ));
+
+      // sorts the line data whose capacity are equal according to the name
+      const sortFunction = (obj) => {
+        obj.sort(function(a, b) {
```

Comment:
Nit: can we change the name to something meaningful, all over.

Reply:
ack